### DISCUSSION TITLE
Support for `repeated bytes` fields

### DISCUSSION THREAD

**Discussion by KerstinKeller** [foxglove/community#294] [Q&A]
Foxglove cannot handle `repeated bytes` fields (as defined in a protobuf schema), which is ok.
However, if any channel within a mcap measurement contains a `repeated bytes` field in its channels, the complete measurement cannot be loaded.

It would be preferrable that all channels except the ones containing `repeated byte` fields are loaded. 
Or even better, all channels are loaded but `repeated bytes` fields are not populated.
Or the very best case, that Studio does support repeated bytes 😄 

![image](https://user-images.githubusercontent.com/13848742/172791384-a6dd6ac5-8b06-4ed0-a402-e3534552c3e3.png)

**Comment by defunctzombie** [comment]
@jtbandes adding to the BYOD list so we can find an appropriate mediation. I agree that at minimum we should fail that one channel rather than all channels.

**Comment by jtbandes** [comment]
A proper fix to actually support repeated bytes would also be related to https://github.com/foxglove/community/issues/206 / https://github.com/foxglove/studio/pull/3091

**Comment by jtbandes** [comment]
Updated the app so it continues parsing channels after an error is encountered in one channel (#3733). This ticket can now track support for `repeated bytes`

**Comment by defunctzombie** [comment]
@jtbandes Do you still plan to tackle this or relinquish assignment and let it float to the backlog?

**Comment by jtbandes** [comment]
No current plans, haven't seen much demand.
